pkgs/os-specific/linux/libcap/default.nix

```diff
@@ -41,10 +43,12 @@ stdenv.mkDerivation rec {
       --replace 'man_prefix=$(prefix)' "man_prefix=$doc"
   '';
 
-  installFlags = [ "RAISE_SETFCAP=no" ];
+  patches = lib.optional static ./no-shared-lib.patch;
```

Comment:
`patches` should ideally go after `src`, see https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887.